frontend/packages/pipelines-plugin/src/components/shared/workspaces/__tests__/WorkspaceDefinitionList.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { ShallowWrapper, shallow } from 'enzyme';
+import WorkspaceDefinitionList, { WorkspaceDefinitionListProps } from '../WorkspaceDefinitionList';
+import { pipelineTestData, PipelineExampleNames } from '../../../../test-data/pipeline-data';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+describe('WorkspaceDefinitionList', () => {
+  let workspaceDefinitionListWrapper: ShallowWrapper<WorkspaceDefinitionListProps>;
+  let workspaceDefinitionListWrapperProps: WorkspaceDefinitionListProps;
+
+  it('Should not render root dl if no workspaces are found', () => {
+    workspaceDefinitionListWrapperProps = {
+      workspaces: null,
+    };
+    workspaceDefinitionListWrapper = shallow(
+      <WorkspaceDefinitionList {...workspaceDefinitionListWrapperProps} />,
+    );
+    expect(workspaceDefinitionListWrapper.find('dl').exists()).toBe(false);
```

Comment:
I think based on Christian's response, we want this:

```suggestion
    workspaceDefinitionListWrapper = shallow(
      <WorkspaceDefinitionList {...workspaceDefinitionListWrapperProps} />,
    );
    expect(workspaceDefinitionListWrapper.isEmptyRender()).toBe(true);
```

Ref: https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/isEmptyRender.html

Reply:
updated